js/components/feed-input.tsx

```diff
@@ -106,41 +108,56 @@ class FeedInput extends React.Component<IFeedInputProps, IFeedInputState> {
 
         const FileUpload: typeof fileUpload = manywho.component.getByName(registeredComponents.FILE_UPLOAD);
 
-        let fileUpload = null;
+        let fileUploadComponent = null;
 
         if (this.props.isAttachmentsEnabled) {
 
             const fileUploadProps = {
                 flowKey: this.props.flowKey,
                 multiple: true,
-                upload: manywho.social.attachFiles,
+                upload: (flowKey: string, _: FormData, onProgress: EventListenerOrEventListenerObject, files: File[], fileDataRequest: any) => {
+                    // Construct some form data, for backwards compatibility
+                    const formData = new FormData();
+                    files && files.forEach((file, i) => {
+                        formData.append(`FileData${i}`, file);
+                    });
+
+                    manywho.social.attachFiles(flowKey, formData, onProgress)
```

Comment:
What happens if this request fails? How is the error surfaced in the UI? I'm not sure if UI Core takes care of that or not. Might be worth checking.

Reply:
@jamesbrattBoomi Poked around ui-core regarding this...(social.ts > ajax.ts > connection.ts > upload). Don't think an error would be surfaced to the UI. If this fails it looks to me like the failure is just logged (using loglevel).

Reply:
I left a note in the PR description about what happens if this component (or the feed component) errors - it bricks the entire page, except for any navigation elements :(

Reply:
@DaliborDell There is the Model.addNotification function that hangs off of ui-core. I wonder if you could call it from a catch block here to prevent the flow crashing?